src/deck.jsx

```diff
@@ -49,37 +50,56 @@ class Deck extends React.Component {
   }
   _handleKeyPress(e) {
     const event = window.event ? window.event : e;
-    if (event.keyCode === 37) {
+    if (event.keyCode === 37) { // left arrow
       this._prevSlide();
     }
-    if (event.keyCode === 39) {
+    if (event.keyCode === 39) { // right arrow
       this._nextSlide();
     }
+    if (event.keyCode === 79) { // o
+      this._toggleOverviewMode();
+    }
+    if (event.keyCode === 80) { // o
+      this._togglePresenterMode();
+    }
+  }
+  _toggleOverviewMode() {
```

Comment:
We should probably clear the flux stores when going into these modes. Sometime you have to hit the right arrow double as many times to get the fragments to start appearing.